tools/cli/wskaction.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
 
     def getItemSpecificCommands(self, parser, props):
         subcmd = parser.add_parser('create', help='create new action')
+        subcmd.add_argument('--type', help='the type of version of the runtime')
```

Comment:
Any reason why it's called `type` as opposed to the already-used-internally `kind`?